packages/hub/db/migrations/20211207151150639_sent-push-notifications.js

```diff
@@ -0,0 +1,23 @@
+const SENT_PUSH_NOTIFICATIONS_TABLE = 'sent_push_notifications';
+
+exports.up = (pgm) => {
+  // TODO: may be useful to add a message id field for debug purposes
+  // The message id is returned by firebase messaging
```

Comment:
https://firebase.google.com/docs/cloud-messaging/send-message#send-messages-to-specific-devices